src/style.css

```diff
@@ -2,0 +1,90 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+/* stylelint-disable font-family-name-quotes */
+@font-face {
+  font-family: 'Avenir';
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: 'Avenir';
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+@font-face {
+  font-family: 'Avenir';
```

Comment:
Again, consider adding a generic fallback font to the 'Avenir' font-family to ensure text is displayed correctly if the custom font fails to load.